src/main/java/com/pinterest/secor/uploader/Uploader.java

```diff
@@ -171,15 +176,21 @@ protected void uploadFiles(TopicPartition topicPartition) throws Exception {
                 if (mDeterministicUploadPolicyTracker != null) {
                     mDeterministicUploadPolicyTracker.reset(topicPartition);
                 }
-                mZookeeperConnector.setCommittedOffsetCount(topicPartition, lastSeenOffset + 1);
```

Comment:
Why this line is removed?

Reply:
It wasn't. It was wrapped into `if` statement and now currently in lines 185-189.